apache/superset
# chore: Select component refactoring - FilterControl - Iteration 5

### SUMMARY
It replaces the react-select Select component with the Antdesign one in the FilterControl components.


### BEFORE/AFTER

https://user-images.githubusercontent.com/60598000/126142095-30fa08a9-beb2-4098-95f0-753f2fcbd64c.mp4

https://user-images.githubusercontent.com/60598000/126144838-20b68e04-2bb4-4acb-9f44-27e2dc8712cf.mp4

### BEFORE/AFTER

https://user-images.githubusercontent.com/60598000/126142234-93cb7da6-da2e-498b-9118-9e705402bd1f.mp4

https://user-images.githubusercontent.com/60598000/126144743-79a2adcb-859b-4a97-913b-3b8b0e98017b.mp4

### TESTING INSTRUCTIONS
1. Open a chart in Explore
2. Add Filters using the SIMPLE tab options and observe the behavior
3. Add Filters using the CUSTOM SQL options and observe the behavior

### ADDITIONAL INFORMATION
